src/services/useAwsPlaceService.ts

```diff
@@ -83,6 +84,15 @@ const useAwsPlaceService = () => {
 				};
 
 				return await locationClient?.searchPlaceIndexForPosition(params).promise();
+			},
+			getNLPlacesByText: async (Text: string) => {
+				const response = await fetch(`${NLP_BASE_URL}/places/ask?` + new URLSearchParams({ Text: Text }), {
+					method: "GET",
+					headers: {
+						"x-api-key": "lBrENB1FDv5cDfT3cHQSd9qKJuut3hIU4ZdCeeO8"
```

Comment:
Also, lets remove this and set this again as a `VITE_` configuration. We do not want traffic to this before we release.